src/model/documentselection.js

```diff
@@ -388,6 +392,27 @@ export default class DocumentSelection {
 		return this._selection._getStoredAttributes();
 	}
 
+	/**
+	 * Temporarily and partially disables default gravity behaviour that tries to get attributes from nodes surrounding the caret.
```

Comment:
"behavior" in AmE.

And what does "partially" mean here?

Besides, it'd be good to make some minimal introduction here to what is that gravity.

> Temporarily changes the gravity of the selection from left to right. The gravity defines from which direction the selection inherits its attributes. If it's the default left gravity, the selection (after being moved by the user) inherits attributes from its left hand side. This method allows to temporarily override this behavior by forcing the gravity to the right.

Reply:
I can see that you wrote more thorough docs in the writer. Still, you can update these to what I described and you should link, in such a case, to the writer. Make it easier for developers to find the docs.

Reply:
> and you should link, in such a case, to the writer

there is a link
```
* @see module:engine/model/writer~Writer#overrideSelectionGravity
```

Reply:
Sorry, I forgot I found it there in the meantime :D